src/fastga_he/models/performances/mission_vector/mission_vector.py

```diff
@@ -71,6 +74,14 @@ def __init__(self, **kwargs):
         # worth while to rerun the non linear guesses
         self._last_mtow = 0.0
 
+        # TODO: Change the service name in FAST-GA so that this is not necessary anymore
+        if RTA_INSTALLED:
+            oad.RegisterSubmodel.active_models["service.geometry.wing"] = (
+                "fastga.submodel.geometry.wing.legacy"
+            )
+
+            _LOGGER.warning("FAST-OAD RTA is installed in this local machine !")
```

Comment:
I'm rethinking the need for putting a logger message here. At least not a warning severity